### ISSUE TITLE
clarify SQL licensing FAQ

### ISSUE THREAD

**Issue by aczechowski** [MicrosoftDocs/SCCMdocs#464]
[Frequently asked questions for Configuration Manager branches and licensing](https://docs.microsoft.com/sccm/core/understand/product-and-licensing-faq#bkmk_sql)
*What are the use rights associated with the SQL technology provided with System Center Configuration Manager?*

Additional questions to address:
- Enterprise vs Standard
- Extra dba stuff like "cmmonitor"
- PowerBI server for external reporting
- Database vs instance vs server

Also investigate how to further clarify: 
- 3rd-party (and even 1st-party) add-ons: partner products, MBAM, and MDT

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c3b9a7fa-9585-a2fb-590e-0b74e128035d
* Version Independent ID: 9aae8e97-de68-c274-b4c6-cd925568c75b
* Content: [Product and Licensing FAQ - Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/understand/product-and-licensing-faq)
* Content Source: [sccm/core/understand/product-and-licensing-faq.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/understand/product-and-licensing-faq.md)
* Service: **unspecified**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by Silencer0001** [comment]
I see this article is updated on 30/04. Does this reflect the full statement of Microsoft after raising your question?
I would like to have an official statement about including the following database to the SQL-Server of ConfigMgr:
     - cmmonitor
     - ConfigMgr Client Health DB by Anders Rodland
     - MDT Database

All the Databases above can be linked to additional functionality for ConfigMgr.
I would generally add features like "PowerBI" and "MBAM" to another (licensed) DB Instance since it's not tightly related to ConfigMgr.

Thanks for the feedback!

**Comment by aczechowski** [comment]
@Silencer0001, see the following statement in [this particular FAQ](https://docs.microsoft.com/sccm/core/understand/product-and-licensing-faq#bkmk_sql):
> If a database for any additional Microsoft or third-party product shares the SQL Server, you must have a separate license for that SQL Server instance.  

At this time, the products that you specify fall into this category.

**Comment by Silencer0001** [comment]
@aczechowski, thanks for the reply. I thought so, but wasn't sure since this feedback item was still "open". Too bad to hear that it doesn't fall under the standard SQL license. Are there any other native Microsoft solutions available to index and optimise the SQL DB without violating the SQL license terms for ConfigMgr?

**Comment by bahtisametcoban** [comment]
@aczechowski I have a database to store dba related information such as logins and sizes and so on. Is it also falls into that category?
My second question is what happens if I create sp,table under system databases? I would like to know if that is a problem for licencing.

**Comment by aczechowski** [comment]
Question from AZSMUG to clarify: customer has Intune through EMS/M365, thus has SCCM. Does that also then include use rights for SQL?

**Comment by aczechowski** [comment]
- Is SQL **Cluster** included? (we only talk about SQL AO)
- Would a full SQL Standard be also included when used for **Secondary sites**?

**Comment by aczechowski** [comment]
See linked #1077, closed as a duplicate of this.

**Comment by JBFulc** [comment]
@aczechowski just to update since applying the product key for SCCM (that I got from the 365 team) and the server being restarted. SQL has now locked me out as it needs a Licence key too. I have recontacted the same team to obtain a SQL 2017 key which they are looking into. I guess you are looking at the same thing?

**Comment by JBFulc** [comment]
@aczechowski I am still unable to use SCCM. As SQL is no longer licensed the SQL services cannot be started.
Whom at Microsoft support should I contact to get hold of a SQL licence key?

**Comment by JBFulc** [comment]
> Question from AZSMUG to clarify: customer has Intune through EMS/M365, thus has SCCM. Does that also then include use rights for SQL?

Is there any clarification on this or do I need to buy a SQL licence separately?
Thanks.

**Comment by aczechowski** [comment]
I don't have clarity on that yet. I know support is working with the licensing team to help unblock you.

**Comment by JBFulc** [comment]
> I don't have clarity on that yet. I know support is working with the licensing team to help unblock you.

Is there any timeline on this? Am I able to contact the support team that are looking into this for me?
Thanks.

**Comment by JBFulc** [comment]
@aczechowski is there any news from the licensing team?

**Comment by faulknerp** [comment]
Any update on whether SQL Enterprise licence is covered please? I asked this before but it was closed as a duplicate of #464...


SQL Enterprise is required for AlwaysOn (https://docs.microsoft.com/en-us/sccm/core/servers/deploy/configure/sql-server-alwayson-for-a-highly-available-site-database).
As this page says approved SQL use rights with SCCM include "SQL Server Always On" does that confirm that SQL Enterprise is covered by the license that comes with SCCM?

**Comment by JBFulc** [comment]
@aczechowski Is there an appropriate department that my CSP should be talking to? As wording seems to be everything when contacting Microsoft support is there anything specific they should mention?

**Comment by aczechowski** [comment]
For my own reference: 

From [licensing terms](https://www.microsoft.com/licensing/product-licensing/products):
> 3.2 SQL Server Technology
> Customer may run any number of *Instances* of any SQL Server database software included in the Product in one *OSE* only for the purpose of supporting that Product and any other Product that includes SQL Server database software.

[Direct link to English product terms doc](http://www.microsoftvolumelicensing.com/Downloader.aspx?documenttype=PT&lang=English)

VLSC lists SQL Standard along with the SCCM current branch download.

With SQL 2016, SQL Reporting Services is a separate download. PowerBI server requires Enterprise edition.

**Comment by anwojcie** [comment]
@aczechowski 
Statement "Right to run management server software and supporting SQL Server Runtime (SQL Server **Standard Edition**)"
from https://www.microsoft.com/en-us/licensing/product-licensing/system-center?activetab=product-licensing/system-center-2016-pivot:primaryr2
was sent to my CX recently by LicenseQ

**Comment by dsolodow** [comment]
Not sure how it would be worded, but a _small_ user database for management tasks/proc (like OlaHollangren) would seem useful.

**Comment by zilvakas** [comment]
SCCM is bought through EMS. Questions: 
 - How to get SQL license key?
 - is it possible to use any version of SQL (I mean 2014/2016, 2017, 2019)?

**Comment by aczechowski** [comment]
@zilvakas, see [this FAQ](https://docs.microsoft.com/configmgr/core/understand/product-and-licensing-faq#bkmk_csp). At least get a support case open for the ConfigMgr key. I haven't yet seen a definitive response from business planning on the process to get SQL keys in this situation, but I'm still checking. 

Regarding version of SQL, see the [supported configuration](https://docs.microsoft.com/configmgr/core/plan-design/configs/support-for-sql-server-versions).

**Comment by aczechowski** [comment]
After much internal discussion, I'm closing this issue. I realize that there are some questions here that are not explicitly answered.

FYI, I did also submit a change to fix related issue #1274.

If you still have further questions, please talk with your Microsoft account manager about your organization's licensing, and review the official [product terms](https://www.microsoftvolumelicensing.com/DocumentSearch.aspx?Mode=3&DocumentTypeId=53).
